packages/core-sdk/resolvers/protocol-resolvers/resolvers/ethereum/src/query/functions/resolveProtocol.ts

```diff
@@ -41,6 +41,12 @@ export function resolveProtocol(input: Input_resolveProtocol): ProtocolResolver_
     return supportedProtocolsMap.get("cream_v1");
   } else if (input.token.name == "SushiBar") {
     return supportedProtocolsMap.get("sushibar_v1");
+  } else if (input.token.name.includes("Smart Token Relay")) {
+    return supportedProtocolsMap.get("bancor_v2.1");
+  } else if (input.token.name.includes("Smart Token Relay")) {
+    return supportedProtocolsMap.get("bancor_v2");
```

Comment:
This is conflicting and can cause issues.

Reply:
I believe that If we can't find good ways to resolve protocol, we should use brute force instead.

Reply:
see response above